content/geoip/geolite2-free-geolocation-data.mdx

```diff
@@ -136,3 +136,17 @@ package the databases with your commercial products without the need for
 attribution. For more information, please [visit the GeoLite2 Commercial
 Redistribution
 page](https://www.maxmind.com/en/geolite2-commercial-redistribution).
+
+## Earn a Commission with your Integration
```

Comment:
We tend to capitalize `Your` in our other headings (e.g. see the `Share Your Tools` on the contribute page), so I think we should be consistent and do it here as well. I also did a [brief search](https://www.elevenwriting.com/blog/is-your-capitalized-in-a-title#:~:text=The%20reason%20you%20should%20always,the%20word%20doesn't%20matter!) to confirm.